tests/console/machinery.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use testapi;
+
+sub run() {
+    select_console 'root-console';
+
+    # add machinery repository from ymp file
+    assert_script_run "yast OneClickInstallCLI prepareinstall url=http://machinery-project.org/machinery.ymp targetfile=/tmp/local.ymp";
+    assert_script_run "yast OneClickInstallCLI doinstall instructionsfile=/tmp/local.ymp";
+
+    assert_screen ['Import Untrusted GnuPG Key'], 60;
```

Comment:
That's an unconventional tag name. It will work, but we already have similar tag `import-untrusted-gpg-key`.
Also you don't have to pass it as arrayref `[]`, regular string is ok in case of only one tag:
```perl
assert_screen 'import-untrusted-gpg-key', 60;
```